2020_keychain_PIC/ee_tut/ee_tut.sch

```diff
@@ -0,0 +1,454 @@
+EESchema Schematic File Version 4
+EELAYER 26 0
+EELAYER END
+$Descr A4 11693 8268
+encoding utf-8
+Sheet 1 1
+Title ""
+Date ""
+Rev ""
+Comp ""
+Comment1 ""
+Comment2 ""
+Comment3 ""
+Comment4 ""
+$EndDescr
+$Comp
+L New_Library:PIC16LF15313 U1
+U 1 1 5F5409CC
+P 5650 3450
+F 0 "U1" H 5650 3965 50  0000 C CNN
+F 1 "PIC16LF15313" H 5650 3874 50  0000 C CNN
+F 2 "" H 5650 3450 50  0001 C CNN
+F 3 "" H 5650 3450 50  0001 C CNN
+	1    5650 3450
+	1    0    0    -1  
+$EndComp
+$Comp
+L Device:Battery_Cell BT1
+U 1 1 5F540ABC
+P 7950 2650
+F 0 "BT1" H 8068 2746 50  0000 L CNN
+F 1 "Battery_Holder" H 8068 2655 50  0000 L CNN
+F 2 "" V 7950 2710 50  0001 C CNN
+F 3 "~" V 7950 2710 50  0001 C CNN
+	1    7950 2650
+	1    0    0    -1  
+$EndComp
+$Comp
+L Device:LED D1
+U 1 1 5F540BBA
+P 6600 4050
+F 0 "D1" V 6638 3933 50  0000 R CNN
+F 1 "LED" V 6547 3933 50  0000 R CNN
+F 2 "" H 6600 4050 50  0001 C CNN
+F 3 "~" H 6600 4050 50  0001 C CNN
+	1    6600 4050
+	0    -1   -1   0   
+$EndComp
+$Comp
+L Device:LED D2
+U 1 1 5F540C46
+P 6950 4050
+F 0 "D2" V 6988 3933 50  0000 R CNN
+F 1 "LED" V 6897 3933 50  0000 R CNN
+F 2 "" H 6950 4050 50  0001 C CNN
+F 3 "~" H 6950 4050 50  0001 C CNN
+	1    6950 4050
+	0    -1   -1   0   
+$EndComp
+$Comp
+L Device:C C1
+U 1 1 5F540CB8
+P 4700 2850
+F 0 "C1" H 4815 2896 50  0000 L CNN
+F 1 "0.1uF" H 4815 2805 50  0000 L CNN
+F 2 "" H 4738 2700 50  0001 C CNN
+F 3 "~" H 4700 2850 50  0001 C CNN
+	1    4700 2850
+	1    0    0    -1  
+$EndComp
+$Comp
+L Switch:SW_Push SW1
+U 1 1 5F540D4F
+P 4100 3350
+F 0 "SW1" H 4100 3635 50  0000 C CNN
+F 1 "SW_Push" H 4100 3544 50  0000 C CNN
+F 2 "" H 4100 3550 50  0001 C CNN
+F 3 "" H 4100 3550 50  0001 C CNN
+	1    4100 3350
+	1    0    0    -1  
+$EndComp
+$Comp
+L Switch:SW_Push SW2
+U 1 1 5F540D97
+P 4100 4000
+F 0 "SW2" H 4100 4285 50  0000 C CNN
+F 1 "SW_Push" H 4100 4194 50  0000 C CNN
+F 2 "" H 4100 4200 50  0001 C CNN
+F 3 "" H 4100 4200 50  0001 C CNN
+	1    4100 4000
+	1    0    0    -1  
+$EndComp
+$Comp
+L Connector:Conn_01x05_Female J1
+U 1 1 5F540E8E
+P 8900 3400
+F 0 "J1" H 8927 3426 50  0000 L CNN
+F 1 "Programming_Connector" H 8927 3335 50  0000 L CNN
+F 2 "" H 8900 3400 50  0001 C CNN
+F 3 "~" H 8900 3400 50  0001 C CNN
+	1    8900 3400
+	1    0    0    -1  
+$EndComp
+$Comp
+L Switch:SW_SPST SW3
+U 1 1 5F5411AF
+P 8250 2350
+F 0 "SW3" H 8250 2585 50  0000 C CNN
+F 1 "SW_SPST" H 8250 2494 50  0000 C CNN
+F 2 "" H 8250 2350 50  0001 C CNN
+F 3 "" H 8250 2350 50  0001 C CNN
+	1    8250 2350
+	1    0    0    -1  
+$EndComp
+$Comp
+L Device:R_US R4
+U 1 1 5F54130F
+P 6600 3750
+F 0 "R4" H 6668 3796 50  0000 L CNN
+F 1 "10k" H 6668 3705 50  0000 L CNN
```

Comment:
This seems a bit high for a current limiting resistor on an LED, is the part value intended?

Reply:
lol whoops, went a little over zealous on labeling things as 10K